modules/custom/claim_shield.lua

```diff
@@ -0,0 +1,93 @@
+-----------------------------------
+-- Claim Shield
+-----------------------------------
+require("modules/module_utils")
+require("scripts/globals/msg")
+require("scripts/globals/utils")
+-----------------------------------
+local m = Module:new("claim_shield")
+m:setEnabled(true)
```

Comment:
Should we default to enabled=false for custom modules?

Reply:
Whoops, yes